db/migrate/20161108105410_create_products.rb

```diff
@@ -0,0 +1,17 @@
+class CreateProducts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :products do |t|
+    	
+    	t.string :title
+    	t.boolean :taxable
+    	t.string :featured_image
+    	t.boolean :available
+    	t.string :price
+    	t.integer :grams
+    	t.string :compare_at_price
+    	t.bigint :product_id
```

Comment:
it was updated to `product_code` a few days ago. Would that be fine or should it be changed to `unique_product_id` ?